tests/jeos/build_key.pm

```diff
@@ -14,18 +14,25 @@ use strict;
 use warnings;
 use base "opensusebasetest";
 use testapi;
+use version_utils "is_opensuse";
 
 sub run {
     # check that suse-build-key is installed
     my $zypper_output = script_output('zypper search suse-build-key');
-    die 'suse-build-key is not installed' unless $zypper_output =~ /i\+(.*)suse-build-key/;
 
-    assert_script_run('ls /usr/lib/rpm/gnupg/keys | grep suse_ptf_key.asc');
+    if (is_opensuse) {
+        die 'suse-build-key is not installed' unless $zypper_output =~ /i\+(.*)openSUSE-build-key/;
```

Comment:
It would be:
```die 'openSUSE-build-key is not installed' unless $zypper_output =~ /i\+(.*)openSUSE-build-key/;```

Reply:
Updated.
New validation runs: http://ccret.suse.cz/tests/2912, http://ccret.suse.cz/tests/2913